### Table information

| Variable | Original Name (AADR) | Values |
| :-------- | :-------------------- | :------: |
| Genetic ID | Genetic ID | Unique identifier for each sequencing data |
| Master ID | Master ID | Unique identifier for each individual, one individual can have multiple Genetic IDs |
| Skeletal code | Skeletal code | Unique identifier for the individual skeletal |
| Skeletal element |	 Skeletal element |
| Published Year |	 Year data from this individual was first published [for a present-day individuals we give the data of the data reported here; missing GreenScience 2010 (Vi33.15, Vi33.26), Olalde2018 (I2657), RasmussenNature2010 (Australian)] |
| Publication |	 Publication |
| Dating method |	 Method for Determining Date; unless otherwise specified, calibrations use 95.4% intervals from OxCal v4.4.2 Bronk Ramsey (2009); r5; Atmospheric data from Reimer et al (2020) |
| Date Mean in BP |	 Date mean in BP in years before 1950 CE [OxCal mu for a direct radiocarbon date, and average of range for a contextual date] |
| Date SD in BP |	 Date standard deviation in BP [OxCal sigma for a direct radiocarbon date, and standard deviation of the uniform distribution between the two bounds for a contextual date] |
| Full date |	 Full Date One of two formats. (Format 1) 95.4% CI calibrated radiocarbon age (Conventional Radiocarbon Age BP, Lab number) e.g. 2624-2350 calBCE (3990Â±40 BP, Ua-35016). (Format 2) Archaeological context range, e.g. 2500-1700 BCE |
| Age at Death |	 Age at Death from physical anthropology |
| Group ID |	 Group ID |
| Locality |	 Locality |
| Locality ID |	 Not included |
| Correct Locality Name |	 Not included |
| Country |	 Not included |
| Unique Locality Name |	 Not included |
| Political Entity |	 Political Entity |
| Latitude |	 Lat. |
| Longitude |	 Long. |
| Pulldown Strategy |	 Pulldown Strategy |
| Data source |	 Data source |
| No. Libraries |	 No. Libraries |
| 120k coverage |	 1240k coverage (taken from original pulldown where possible) |
| SNPs hit on autosomal (1240k snpset)|	 SNPs hit on autosomal targets (Computed using easystats on 1240k snpset) |
| SNPs hit on autosomal (HO snpset) |	 SNPs hit on autosomal targets (Computed using easystats on HO snpset) |
| Molecular Sex |	 Molecular Sex |
| Family ID |	 Family ID and position within family |
| Y Haplogroup MF |	 Y haplogroup (manual curation in terminal mutation format) |
| Y Haplogroup ISOGG |	 Y haplogroup (manual curation in ISOGG format) |
| mtDNA coverage |	 mtDNA coverage (merged data) |
| mtDNA Haplogroup |	 mtDNA haplogroup if >2x or published |
| mtDNA match |	 mtDNA match to consensus if >2x (merged data) |
| Damage rate |	 Damage rate in first nucleotide on sequences overlapping 1240k targets (merged data) |
| Sex ratio |	 Sex ratio [Y/(Y+X) counts] (merged data) |
| Library type |	 Library type (minus=no.damage.correction, half=damage.retained.at.last.position, plus=damage.fully.corrected, ds=double.stranded.library.preparation, ss=single.stranded.library.preparation) |
| Libraries |	 Libraries |
| Period |	
| Usage Note |	
| DataRef |	
| Reference |	
| qpAdm Pvalue |	
| Steppe |	
| Steppe SE |	
| WHG |	
| WHG SE |	
| EEF |	
| EEF SE |	
| PCA1 |	
| PCA2 |	
| PCA3 |	
| PCA4 |	
| PCA5 |	
| PCA6 |	
| PCA7 |	
| PCA8 |	
| PCA9 |	
| PCA10 |	
| NOT INCLUDED |	 ASSESSMENT |
| NOT INCLUDED |	 ASSESSMENT WARNINGS (Xcontam interval is listed if lower bound is >0.005, \QUESTIONABLE\ if lower bound is 0.01-0.02, \QUESTIONABLE_CRITICAL\ or \FAIL\ if lower bound is >0.02) (mtcontam confidence interval is listed if coverage >2 and upper bound is <0.|
